HW_18/task_2.py

```diff
@@ -0,0 +1,38 @@
+import random
+
+
+class Boss:
+
+    def __init__(self, name: str, company: str):
+        self.id = random.randint(1, 999)
+        self.name = name
+        self.company = company
+        self.workers = []
+
+    @property
+    def add_workers(self):
+        return self.workers
+
+    @add_workers.setter
+    def add_workers(self, worker):
+        self.workers.append(worker)
+
+
+class Worker:
+
+    def __init__(self, name: str, boss: Boss):
+        self.id_ = random.randint(1, 9999)
```

Comment:
Try to avoid `random` in your programs unless it's in requirements to generate something random. Use deterministic approach (approach with `random` can lead to collisions)